docs/output-targets/docs-custom.md

```diff
@@ -55,21 +55,21 @@ The generated docs JSON data will in the type of `JsonDocs` which consists of ma
 | `fileName`    | File name |
 | `filePath`    | File path |
 | `readmePath`    | Readme file path |
-| `usagesDir`    | Usages directory path  |
+| `usagesDir`    | Stencil looks in a directory named `usages/` in the same directory as your component to find usage examples. This holds the full path to that directory. |
 | `encapsulation`    | Component `encapsulation` type. Possible values are `shadow`, `scoped`, `none`  |
 | `tag`    | Component tag described in `.tsx` file  |
 | `readme`    | Component readme file first line content  |
 | `docs`    | Description written in top of `@Component` e.g. /**  Documentation Example */. If no JSDoc is present, default to any manually written text in the component's markdown file. Empty otherwise. |
 | `docsTags`    | Annotations (In the way of JSDoc ) written in `.tsx` file will be collected here   |
 | `overview`    | Description written in top of `@Component` e.g. /**  Documentation Example */ |
-| `usage`    |    |
+| `usage`    | Array of [usage examples](./docs-json.md#usage), written in Markdown files in the `usages/` directory adjacent to your component. |
 | `props`    | Array of component properties information   |
```

Comment:
I had started this work a long time ago, but wasn't able to finish it. Thoughts on updating `props` thru `listeners` to include their decorator in the description, like so? https://github.com/ionic-team/stencil-site/pull/918/files#diff-bbddb338be506fb83391c3cc04b87642f18868a7ba2707bc1eb148d589c69421R74-R80